trp_project_scrum/tests/test_trp_project_scrum.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp.tests.common import TransactionCase
+from openerp.exceptions import AccessError
+import random
+import string
+
+
+class TestTrpProjectScrum(TransactionCase):
```

Comment:
Simplify names by dropping Trp from class and _trp from methods. No danger for clashing with other code.